cfripper/config/config.py

```diff
@@ -192,6 +202,43 @@ def load_rules_config_file(self, rules_config_file: TextIOWrapper):
             logger.exception(f"Failed to read config file: {filename}")
             raise
 
+    def add_filters_from_dir(self, path: str):
+        if not os.path.exists(path):
```

Comment:
Given that cfripper is python >=3.7, I'd recommend using Pathlib instead of os.path.

Reply:
Updated